index.js

```diff
@@ -63,8 +63,9 @@ function plugin(opts) {
 
         debug('adding %s to includes as `%s`', resolvedFilename, name);
 
-        file[name] = included[filename].contents;
-
+        file[name] = included[filename];
+        file[name].contents = included[filename].contents;
```

Comment:
@madeofpeople if `included[filename] = { contents: "blah" }`, then setting `file[name] = included[filename]` is equivalent to `file[name] = { contents: "blah" };`, right?